internal/configuration/sources_test.go

```diff
@@ -0,0 +1 @@
+package configuration
```

Comment:
We don't need this file I guess.

Reply:
Yeah probably left it there in refactoring how we provided sources.

Reply:
Removed.